src/types/index.ts

```diff
@@ -23,6 +22,10 @@ export interface ModuleOptions {
   nonce: boolean;
   removeLoggers?: RemoveOptions | false;
   ssg?: Ssg;
+  /**
+   * enable runtime nitro hooks to configure some options at runtime
+   */
+  runtimeHooks: boolean;
```

Comment:
I think you'd be better off omitting this as it may get confusing and buggy if other runtime hooks are added in the future for no perceived value (that I can think of).

Reply:
:+1: